### PR TITLE
05--lost-yak

### PR DESCRIPTION
**Pull request by jeffworley**
Created the above branch to house our js Blueprint practice activities

### CODE REVIEW

File: APIGymnastics/items.json [hunk 1]
```diff
@@ -1,1685 +0,0 @@
-[
```
Comment (kellymurray) [inline APIGymnastics/items.json:1]: why did you delete your items.json file? or did you just move it?

Comment (jeffworley) [inline APIGymnastics/items.json:1]: I moved it after realizing that I had stored it in the incorrect node-modules folder.  Sorry for any confusion this may have caused.
